projects/plugins/jetpack/modules/comments/comments.php

```diff
@@ -614,13 +614,13 @@ public function add_comment_meta( $comment_id ) {
 	 */
 	public function should_show_subscription_modal() {
 
-		// Not allow it to run on self-hosted or simple sites
-		if ( ! ( new Host() )->is_wpcom_platform() || ( new Host() )->is_wpcom_simple() ) {
+		// Only Automattitians for now
+		if ( ! ( defined( 'AT_PROXIED_REQUEST' ) && AT_PROXIED_REQUEST ) ) {
 			return false;
 		}
 
-		// Only Automattitians for now
-		if ( ! ( defined( 'AT_PROXIED_REQUEST' ) && AT_PROXIED_REQUEST ) ) {
+		// Not allow it to run on self-hosted or simple sites
+		if ( ! ( new Host() )->is_wpcom_platform() || ( new Host() )->is_wpcom_simple() ) {
```

Comment:
Why not using `( new Host() )->is_woa_site()` instead?

Reply:
My goal was to leave two options so we can remove the left one when we release it to self-hosted. But I don't have strong feelings about using the one you suggested.

Reply:
Same here 👍